papis/docmatcher.py

```diff
@@ -37,12 +51,13 @@ class MatcherCallable(Protocol):
     .. automethod:: __call__
     """
 
-    def __call__(self,
-                 document: papis.document.Document,
-                 search: Pattern[str],
-                 match_format: Optional[str] = None,
-                 doc_key: Optional[str] = None,
-                 ) -> Any:
+    def __call__(
+        self,
+        document: papis.document.Document,
+        search: Pattern[str],
+        match_format: Optional[str] = None,
+        doc_key: Optional[str] = None,
+    ) -> Any:
```

Comment:
Using black? There are a few unrelated changes like this and it would be helpful to revert them.

Reply:
Yes, I ran black on this. Will revert.